aiida/manage/configuration/config.py

```diff
@@ -8,13 +8,26 @@
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
 """Module that defines the configuration file of an AiiDA instance and functions to create and load it."""
+from __future__ import annotations
+
 import codecs
 from functools import cache
 import json
 import os
-from typing import Any, Dict, Optional, Sequence, Tuple
+from typing import Any, Dict, List, Optional, Sequence, Tuple
```

Comment:
I believe `t.Sequence`is deprecated since 3.9 and should be imported from `collections.abc`. Same for `Dict`, `Optional`, `List`, `Tuple`. But that's I guess for another PR.

Reply:
Ah, interesting. Maybe you could add this comment at the top of the file.

Reply:
Done